packages/core/src/tools/create-file-stasher.js

```diff
@@ -86,12 +86,14 @@ const createFileStasher = (input) => {
       return ZapierPromise.reject(new Error('rpc is not available'));
     }
 
-    const isRunningOnHydrator =
-      _.get(input, '_zapier.event.method', '').indexOf('hydrators.') === 0;
-    const isRunningOnCreate =
-      _.get(input, '_zapier.event.method', '').indexOf('creates.') === 0;
-
-    if (!isRunningOnHydrator && !isRunningOnCreate) {
+    const method = _.get(input, ['_zapier', 'event', 'method'], '');
+    const isValidSource =
+      method.startsWith('hydrators.') ||
+      method.startsWith('creates.') ||
+      // key regex from KeySchema
+      method.match(/^resources\.[a-zA-Z]+[a-zA-Z0-9_]*\.create\./);
```

Comment:
Seems the `+` is unnecessary?

```suggestion
      method.match(/^resources\.[a-zA-Z][a-zA-Z0-9_]*\.create\./);
```

Reply:
sure! I pulled it from the `KeySchema`. it's been that way for ages. 😅 

Updated!